spec/bolt/project_manager/config_migrator_spec.rb

```diff
@@ -2,73 +2,113 @@
 
 require 'pathname'
 require 'spec_helper'
+require 'bolt_spec/project'
 require 'bolt/project_manager/config_migrator'
 
 describe Bolt::ProjectManager::ConfigMigrator do
+  include BoltSpec::Project
+
   def migrate
     migrator.migrate(config_file, project_file, inventory_file, backup_dir)
   end
 
+  let(:inventory_file) { project_dir + 'inventory.yaml' }
```

Comment:
Not alphabetized...why the order change? (Just curious, I don't mind it)

Reply:
Moved them temporarily while I was updating the tests and just copy/pasted them back into the wrong spot. :P